Packs/CommonScripts/Scripts/ParseEmailFilesV2/ParseEmailFilesV2.py

```diff
@@ -0,0 +1,126 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+from parse_emails.parse_emails import EmailParser
+
+
+def data_to_md(email_data, email_file_name=None, parent_email_file=None, print_only_headers=False) -> str:
+    """
+    create Markdown with the data.
+
+    Args:
+      email_data (dict): all the email data.
+      email_file_name (str): the email file name.
+      parent_email_file (str): the parent email file name (for attachment mail).
+      print_only_headers (bool): Whether to only the headers.
+
+    Returns:
+      str: the parsed Markdown
+
+    """
+    if email_data is None:
+        return 'No data extracted from email'
+
+    md = u"### Results:\n"
+    if email_file_name:
+        md = f"### {email_file_name}\n"
+
+    if print_only_headers:
+        return tableToMarkdown(f"Email Headers: {email_file_name}", email_data.get('HeadersMap'))
+
+    if parent_email_file:
+        md += f"### Containing email: {parent_email_file}\n"
+
+    md += u"* {0}:\t{1}\n".format('From', email_data.get('From') or "")
+    md += u"* {0}:\t{1}\n".format('To', email_data.get('To') or "")
+    md += u"* {0}:\t{1}\n".format('CC', email_data.get('CC') or "")
+    md += u"* {0}:\t{1}\n".format('Subject', email_data.get('Subject') or "")
+    if email_data.get('Text'):
+        text = email_data['Text'].replace('<', '[').replace('>', ']')
+        md += u"* {0}:\t{1}\n".format('Body/Text', text or "")
+    if email_data.get('HTML'):
+        md += u"* {0}:\t{1}\n".format('Body/HTML', email_data['HTML'] or "")
+
+    md += u"* {0}:\t{1}\n".format('Attachments', email_data.get('Attachments') or "")
+    md += u"\n\n" + tableToMarkdown('HeadersMap', email_data.get('HeadersMap'))
```

Comment:
This is exactly as being done today right?

Reply:
yes